include/boost/property_tree/json_parser/detail/parser.hpp

```diff
@@ -214,7 +215,8 @@ namespace boost { namespace property_tree {
         void process_codepoint(Sentinel end, EncodingErrorFn error_fn) {
             encoding.transcode_codepoint(cur, end,
                 boost::bind(&Callbacks::on_code_unit,
-                            boost::ref(callbacks), _1),
+                            boost::ref(callbacks),
+                            boost::placeholders::_1),
```

Comment:
one might want this part as explicit as here, since some user code might have `BOOST_BIND_NO_PLACEHOLDERS` defined.

same in the line below.

Reply:
I use libboost1.63 and met the same problem today. After using the above patch, there is still an building error: ‘boost::placeholders’ has not been declared. I solved it by adding the follwing including file: #include <boost/bind/placeholders.hpp>

Reply:
great to hear it helps you!

note that this patch also adds the include you mention include :)